tests/nested/manual/install-min-size/task.yaml

```diff
@@ -47,7 +47,7 @@ execute: |
   boot_id=$(tests.nested boot-id)
 
   # Install new gadget
-  remote.exec "sudo snap install --dangerous pc.snap"
+  remote.exec "sudo snap install --dangerous pc.snap" || [ "$?" -eq 255 ]
```

Comment:
Are there more places that does this kind of thing in nested tests, where this could be applied?

Reply:
Very likely in other tests yes.